### PR TITLE
landingpage redesign

### PR DESCRIPTION
**Pull request by adisve**

### CODE REVIEW

File: client/src/components/landingPage/LandingPage.css [hunk 1]
```diff
@@ -1,45 +1,33 @@
...
+    font-size: 1.2rem;
+    width: 33rem;
+    font-weight: 500;
 }
 
-#landing-page-text{
-    width: 50%;
-    height: fit-content;
-    text-overflow: ellipsis;
+.get-started-btn {
```
Comment (adisve) [inline client/src/components/landingPage/LandingPage.css:23]: Might be a nicer way to do this

File: client/src/components/footer/Footer.css [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #footer {
     box-shadow: none;
-    color: #353e54;
+    background-color: transparent !important;
```
Comment (adisve) [inline client/src/components/footer/Footer.css:3]: This line is inconsequential for now

**Comment by PsychicPlatypus** [comment]
Please merge this when you have the time

**Review by Liz0001 — APPROVED** [review]
Looks great :D
should we also make the logo clickable?

**Comment by adisve** [comment]
> Looks great :D
> 
> should we also make the logo clickable?

Yes, we can do that!

**Pull request merged**